gito/commands/gh_post_review_comment.py

```
@@ -15,16 +15,19 @@
 from ..project_config import ProjectConfig
 
 
-@app.command(help="Leave a GitHub PR comment with the review.")
-def github_comment(
+@app.command(name="github-comment", help="Leave a GitHub PR comment with the review.")
+def post_github_cr_comment(
```

Pylint (3.3.7)

[refactor] Too many branches (13/12)

(R0912)